include/solid_solver.h

```diff
@@ -163,6 +163,8 @@ namespace Solid
      */
     mutable std::vector<std::vector<Vector<double>>> strain, stress;
 
+    mutable Vector <double> cellwise_sxx, cellwise_sxy,cellwise_syy;
```

Comment:
Should be compatible with 3D, i.e.,  when ``dim==3`` there is z direction.

Reply:
Fixed